theano/scalar/basic.py

```diff
@@ -604,12 +604,11 @@ def __sub__(self, other):
     def __mul__(self, other):
         return mul(self, other)
 
-    if PY3:
-        def __truediv__(self, other):
-            return div_proxy(self, other)
-    else:
-        def __div__(self, other):
-            return div_proxy(self, other)
```

Comment:
@nouiz Is that actually correct? What happens if someone calls `a / b` on int tensors, from outside of Theano, without having imported `division` from `__future__`?